response_data/answer_to_file.json

```diff
@@ -0,0 +1,45 @@
+{
+   "After both the Mexican-American War of 1846-48 and California\u2019s establishment as a state in 1850, land grant ownership required validation. The same year, 1867, Ram\u00f3n Rodr\u00edguez and Francisco Alviso had thier ownership validated they sold it to James Archibald.": "speech_sample_0.wav",
```

Comment:
California\u2019s ?

Reply:
Would like to know if these \u___ escaped sequences are intended to be included, I've also marked the other places where they pop up

Reply:
that would make sense, wonder why its in here as the codes instead of the character though

Reply:
It's the Unicode version - we probably want to be using the ascii one, but it might not matter.